core/src/main/java/io/questdb/cairo/TableUtils.java

```diff
@@ -821,16 +821,11 @@ public static void validate(
                 }
 
                 CharSequence name = metaMem.getStr(offset);
-
                 if (getColumnType(metaMem, i) < 0 || nameIndex.put(name, denseCount++)) {
                     offset += Vm.getStorageLength(name);
                 } else {
                     throw validationException(metaMem).put("Duplicate column: ").put(name).put(" at [").put(i).put(']');
                 }
-
-                if (offset >= memSize) {
```

Comment:
This may be useful to check that last column name does not go beyond file size

Reply:
that is already checked at line 815.
could not construct a test where this exception is thrown so removed it.